### PR TITLE
remove code-analysis diagnostics

### PR DESCRIPTION
**Pull request by JDanielSmith**
Remove diagnostics from both **CodeQL** and Visual Studio.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,9 +22,8 @@ Problems and Configurations
 
   - Pass -DCMAKE_INSTALL_PREFIX to set the install location.
 
-  - Python and C++ bindings are built. Just make sure the relevant tools
-    are on your PATH.  Java and MATLAB bindings are now in the **archive** directory
-    and no longer built.
+  - Java, MATLAB, Python, C++ bindings all build. Just make sure the relevant tools
```
Comment (bradh) [inline README.md:25]: Looks like a regression.

**Review by bradh — COMMENTED** [review]
On first look, perhaps there are things in this change you didn't want to include.

File: README.md (hunk 1 above)
Comment (JDanielSmith) [reply README.md:25]: Nope, that's intentional; the Java and MATLAB bindings "haven't been touched in years."

Comment (bradh) [reply README.md:25]: I agree, but the change doesn't.

The original (current) text says Java and Matlab are "removed / archived", but the new text says they're all fine.

**Pull request closed without merging**